index.js

```diff
@@ -37,7 +51,7 @@ Babelify.prototype._flush = function (callback) {
 
 Babelify.configure = function (opts) {
   opts = assign({}, opts);
-  var extensions = opts.extensions ? babel.util.arrayify(opts.extensions) : null;
+  var extensions = opts.extensions ? arrayify(opts.extensions) : null;
```

Comment:
Do we know if there is a case where this is strictly necessary? It would be much easier to just enforce that people pass the option as an array in the first place. If the CLI is the concern, are there places where browserify won't convert it from a comma list to an array for us?